### DISCUSSION TITLE
Enabled for VAE (always tiled) in the Never OOM integrated section does not work. (at least for me).

### DISCUSSION THREAD

**Discussion by Torcho12** [Panchovix/stable-diffusion-webui-reForge#65] [Q&A] [answered]
Enabled for VAE (always tiled) in the Never OOM integrated section does not work. (at least for me).

p.s with the recent commits in the dev upstream the generation time got longer. For me it is + 15 seconds. (probably a temporary thing)

**Comment by Panchovix** [comment, marked as answer]
Gonna check about that setting, I think after latest VAE upstream updates it may stopped working as expected.

For the performance thing, if you have an NVIDIA card, use --vae-in-fp16 or --vae-in-bf16, since by default now uses float32 which could be slower.

**Reply by Torcho12** [reply]
Thank you!

**Reply by Panchovix** [reply]
It should be fixed now (for both issues), if you want to check.

**Reply by Torcho12** [reply]
It is feels better now. Thank you!

**Reply by Moussack** [reply]
do we still need --vae-in-fp16 or  --vae-in-bf16 ???

**Reply by Horxcodex** [reply]
the VAE still not tiled,i don't see the tiled progress bar in my terminal. I used to see that at the end of img generation.
tried --vae-in-fp16 or --vae-in-bf16 and got cuda oom error. If i dont use any of them i got no error.

**Reply by Panchovix** [reply]
@Moussack shouldn't be needed
@Horxcodex I will try to check when I can!
